php/src/php/Operation.cpp

```diff
@@ -383,15 +383,15 @@ IcePHP::OperationI::getArgInfo(zend_internal_arg_info& arg, const ParamInfoPtr&
     {
         zend_internal_arg_info ai[] =
         {
-            ZEND_ARG_ARRAY_INFO(pass_by_ref, 0, allow_null)
+            ZEND_ARG_ARRAY_INFO((uint32_t)pass_by_ref, (uint32_t)0, (uint32_t)allow_null)
```

Comment:
I think we usually prefer static cast over c-style casts.

Reply:
right I will fix